cli/integration/requirements.txt

```diff
@@ -0,0 +1,5 @@
+requests==2.20.0
+retrying==1.3.3
+pip==9.0.1; python_version >= '3.6'
```

Comment:
Should we settle at python version 3.7?

Reply:
Is there an advantage to disallowing 3.6?

Reply:
Just that we are using the latest known version. Not a big deal either way.